packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarGridList.js

```diff
@@ -51,20 +48,57 @@ const BpkScrollableCalendarGridList = props => {
   const monthsCount = DateUtils.differenceInCalendarMonths(endDate, startDate);
   const months = getMonthsArray(startDate, monthsCount);
 
-  return (
-    <div className={classNames}>
-      <div className={getClassName('bpk-scrollable-calendar-grid-list__strip')}>
-        {months.map((month, index) => (
+  const cache = new CellMeasurerCache({
+    fixedWidth: true,
+    defaultHeight: 276, // most common height (in px) of BpkScrollableCalendarGrid
+  });
+
+  /* eslint-disable react/prop-types */
```

Comment:
We don't disable rules this wholesale in Backpack. If we want to disable it it should be done using `eslint-disable-line` and be paired with a comment explaining why the rule is disabled.